build.gradle

```diff
@@ -104,9 +105,8 @@ project.snapshot.configure { finalizedBy printAllReleasedArtifacts }
 subprojects {
     apply plugin: 'java-library'
```

Comment:
How about we add explicit dep on jar task for control-plane project here in `subprojects` block instead of adding to individual projects.

Reply:
This is a specific handling between control-plane-core etc. and it feels a bit too generic to add these rules for all sub-proj?